src/pages/ExamsPage/index.jsx

```diff
@@ -1,35 +1,53 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 
 import { Tabs, Tab, Container } from '@edx/paragon';
 import { useIntl } from '@edx/frontend-platform/i18n';
 
 import messages from './messages';
-import { useExamsData, useInitializeExamsPage } from './hooks';
-import ExamList from './components/ExamList';
+import {
+  useExamsData, useInitializeExamsPage, useFetchExamAttempts, useExamAttemptsData,
+} from './hooks';
 import AttemptList from './components/AttemptList';
+import ExamList from './components/ExamList';
 import ExternalReviewDashboard from './components/ExternalReviewDashboard';
 
 const ExamsPage = ({ courseId }) => {
   useInitializeExamsPage(courseId);
   const { formatMessage } = useIntl();
   const {
+    currentExam,
     examsList,
     isLoading,
   } = useExamsData();
+  const {
+    attemptsList,
+  } = useExamAttemptsData();
+  /*   eslint-disable react-hooks/exhaustive-deps */
```

Comment:
@ilee2u 
Been thinking about a way around this eslint rule and I've realized something. Maybe this doesn't need to be a useEffect at all. Once we have the exam selector we'll probably have a handler for `handleSelectExam()` or something similar. We should just call `fetchExamAttempts` inside that function. That would be better anyway since otherwise it's select an exam > handleSelectExam() > render with new currentExam > useEffect trigger > render again > fetchExamAttempt > etc. No need for that additional render/useEffect step long long term.

This works for now to test in absence of that feature tho.